war/src/main/java/io/jenkins/pluginhealth/scoring/schedule/ProbeEngineScheduler.java

```diff
@@ -42,9 +42,9 @@ public ProbeEngineScheduler(ProbeEngine probeEngine, ScoringEngine scoringEngine
         this.scoringEngine = scoringEngine;
     }
 
-    @Scheduled(cron = "${app.cron.probe-engine}", zone = "UTC")
+   @Scheduled(cron = "${app.cron.probe-engine}", zone = "UTC")
     public void run() throws IOException {
-        probeEngine.run();
+     //   probeEngine.run();
         scoringEngine.run();
```

Comment:
If you want to change the probe result stored in the database, you need the `probeEngine` to be ran.